src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
random should be `private final` class-level field, because we want to create it's instance once and generate random numbers many times with it's help
```suggestion
        int index = random.nextInt(Color.values().length);
```